diff --git a/dist/index.js b/dist/index.js
index d5bcd3d5366d371269a966ee7d2526758fc70682..b5de19ac9cd4ef8a44ccb5bcc09a9f4b7725429a 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -13990,17 +13990,17 @@ function getWorkerConfig(configPath, env2, opts) {
   if (!config.main) {
     throw new Error(missingFieldErrorMessage(`'main'`, configPath, env2));
   }
-  const mainStat = fs4.statSync(config.main, { throwIfNoEntry: false });
-  if (!mainStat) {
-    throw new Error(
-      `The provided Wrangler config main field (${config.main}) doesn't point to an existing file`
-    );
-  }
-  if (mainStat.isDirectory()) {
-    throw new Error(
-      `The provided Wrangler config main field (${config.main}) points to a directory, it needs to point to a file instead`
-    );
-  }
+  // const mainStat = fs4.statSync(config.main, { throwIfNoEntry: false });
+  // if (!mainStat) {
+  //   throw new Error(
+  //     `The provided Wrangler config main field (${config.main}) doesn't point to an existing file`
+  //   );
+  // }
+  // if (mainStat.isDirectory()) {
+  //   throw new Error(
+  //     `The provided Wrangler config main field (${config.main}) points to a directory, it needs to point to a file instead`
+  //   );
+  // }
   return {
     type: "worker",
     raw,
